index.html

```diff
@@ -3,121 +3,125 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Some Student</title>
+  <title>Kara Fulgum</title>
 
+  <link rel="stylesheet" href="./styles/styles.css">
```

Comment:
Because CSS treats later code as overrides, stylesheets that we write should always come after third party libraries like google fonts or font-awesome.

This is causing the icons to not center in the student info area